Tooltips added, Show only legendaries filter added

karimsah opened the pull request:
- SetSummary : tooltips added for CraftNonGoldenDustRequired and
  CraftGoldenDustRequired
- Replaced ToggleSwitches with Checkboxes
- Added a "Show only legendaries" filter
- Added MissingDesiredAmount in a tooltip

karimsah committed:
Tooltips added

SetSummary : tooltips added for CraftNonGoldenDustRequired and
CraftGoldenDustRequired

karimsah committed:
Merge remote-tracking branch 'refs/remotes/ko-vasilev/master'

karimsah committed:
Replaced ToggleSwitches with Checkboxes

karimsah committed:
Added a "Show only legendaries" filter

karimsah committed:
Added MissingDesiredAmount in a tooltip

Ansem93 commented:
Can you add the possibility to filter cards by searching "common/rare/epic/legendary"?

karimsah commented:
Sure, wouldn't you prefer checkboxes rather than typing it in search filter ?

Ansem93 commented:
I think typing is better, because checkboxe use a lot of space, and it's 
also the same method of the game :)

Il 27/11/2015 08:36, karimsah ha scritto:

> Sure, wouldn't you prefer checkboxes rather than typing it in search 
> filter ?
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/ko-vasilev/Hearthstone-Collection-Tracker/pull/42#issuecomment-160059785.

karimsah commented:
As you wish, then i 'll remove "Show only legendaries" checkbox.
I should have some time next week.

karimsah committed:
Revert "Replaced ToggleSwitches with Checkboxes"

This reverts commit 7edafcc91a69aae7fdeccee236431a5dba019cd0.

Conflicts:
	Hearthstone Collection Tracker/MainWindow.xaml

karimsah committed:
Revert only legendary switch

karimsah committed:
Merge remote-tracking branch 'refs/remotes/ko-vasilev/master'

karimsah committed:
ToggleSwitches are back

karimsah committed:
Added filter by rarity in search filter

Typing common, rare, epic or legendary in search filter does the job.
Modified CardsFilter function for that.

karimsah commented:
Done

Ansem93 commented:
thanks a lot!  Hope to see the new release soon ^^

karimsah commented:
I'll build a release tonight, so you can enjoy it sooner.

Ansem93 commented:
thank you very much!

karimsah commented:
Released, don't hesitate and give feedback

ko-vasilev commented:
Really sorry, I've been really busy for like 2 month, will review and see if some changes are needed this weekend.

karimsah commented:
Haha, no worries, good to have news from you :+1:

karimsah commented:
Glad you merged, for now i have no more ideas,
my thanks for your work !

ko-vasilev commented:
No problem, it needed some tweaks to the code tbh. You can check those out!

Thanks for the contribution :)
